WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -4072,6 +4065,71 @@ function WeakAuras.PickDisplayMultiple(id)
   frame:PickDisplayMultiple(id);
 end
 
+function WeakAuras.PickDisplayMultipleShift(target)
+  if (frame.pickedDisplay) then
+    -- get first aura selected
+    local first;
+    if (WeakAuras.IsPickedMultiple()) then
+      first = tempGroup.controlledChildren[1];
```

Comment:
I think using the last one here is actually better. The controlledChildren of tempGroup are in the order in which they were picked, so using the last one uses the last one that was selected, which matches what's normal for shift selection

Reply:
Indeed, i wasn't thinking enough about how shift-selection works with some auras already selected with ctrl. I also need to change frame.PickDisplayBatch to not do any PickDisplay()